frontend/src/components/object/ObjectMetadata.vue

```diff
@@ -14,6 +14,7 @@ import type { Metadata } from '@/types';
 
 // Props
 type Props = {
+  bucketId: string | undefined ;
```

Comment:
I believe this can be changed to optional attribute instead of explicit undefined typing:
```ts
bucketId?: string;
```

Reply:
i forget why but using `bucketId?: string;` shows a linting error